pkg/kubelet/network/cni/cni.go

```diff
@@ -106,10 +106,21 @@ func (plugin *cniNetworkPlugin) SetUpPod(namespace string, name string, id kubec
 	if !ok {
 		return fmt.Errorf("CNI execution called on non-docker runtime")
 	}
-	netns, err := runtime.GetNetNs(id.ContainerID())
+	hostNet, err := runtime.IsUsingHostNetwork(id.ContainerID())
```

Comment:
Why can't we pass in Pod object here which should ideally include the container ID of the infra container as well?